plugin/src/test/scala/com/here/bom/internal/BomReaderSpec.scala

```diff
@@ -70,9 +74,16 @@ class BomReaderSpec extends AnyFlatSpec with Matchers with MockFactory {
 
     val dependency = "com.test" %% "test_module" % bom
     dependency.revision should be("1.0")
+
+    bom.bomDependencies shouldBe Seq("com.test" % "test_module_2.12" % "1.0")
   }
 
-  private def createMockPomFile(groupId: String, artifactId: String, version: String, addParent: Boolean): File = {
+  private def createMockPomFile(
```

Comment:
This is caused by scalafmt, I could revert it if you prefer as this is not related to any of my change.